dialogue-clients/src/main/java/com/palantir/dialogue/clients/ChannelCache.java

```diff
@@ -75,8 +75,14 @@ static ChannelCache createEmptyCache() {
         LIVE_INSTANCES.add(newCache);
 
         int numLiveInstances = LIVE_INSTANCES.size();
-        if ((numLiveInstances > 1 && log.isInfoEnabled()) || log.isDebugEnabled()) {
```

Comment:
I think we want to emulate replacing `numLiveInstances > 1` with `numLiveInstances >= 10` here, but improving it to log at debug level when debug is enabled and events wouldn't be recorded at info. Totally up to you if you want to include that part, always logging at info, but using `isDebugEnabled` as a sentinel (as before) is fine with me as well.

Reply:
How about this update?